File: GotravelDesktop/src/main/java/com/gotravel/Model/Direccion.java
package com.gotravel.Model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Direccion {

    private int id;
    private String linea1;
    private String linea2;
    private String ciudad;
    private String estado;
    private String pais;
    private String cp;

    public Direccion(String linea1, String linea2, String ciudad, String estado, String pais, String cp) {
        this.linea1 = linea1;
        this.linea2 = linea2;
        this.ciudad = ciudad;
        this.estado = estado;
        this.pais = pais;
        this.cp = cp;
    }
}

File: GoTravelMobile/app/src/main/java/com/gotravel/mobile/data/model/Direccion.kt
package com.gotravel.mobile.data.model

import kotlinx.serialization.Serializable
import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.Setter

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Serializable
data class Direccion (
    val id: Int? = null,
    val linea1: String,
    val linea2: String?,
    val ciudad: String,
    val estado: String,
    val pais: String,
    val cp: String
)